eventtracking/processors/filters.py

```diff
@@ -0,0 +1,44 @@
+"""
+Filter configured through admin panel. Filters out the events that
+do not need to be routed
+"""
+import logging
+
+from eventtracking.processors.exceptions import EventEmissionExit
+
+logger = logging.getLogger(__name__)
+
+
+class RegExFilterProcessor:
+    """
+    Filter the events using regex. The regex list Can be configured
+    through the Admin panel.
+    """
+
+    def __call__(self, event, **kwargs):
+        # To avoid the "Models aren't loaded yet" error,
```

Comment:
Did you try placing this at the top of `eventtracking/django/__init__.py`?
`default_app_config = 'eventtracking.django.apps.EventTrackingConfig'`

Reply:
Tried this now, didn't work. I think since the django tracker initializes the configured engines in the `django/__init__.py` file, it imports the filter even before the app is ready (since all work is being done in the `__init__.py` file.

Although, adding this variable did help in one of the new tests so I'll add it for that.